code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -128,6 +133,7 @@
 	drink_icon = "mojito"
 	drink_name = "Glass of Mojito"
 	drink_desc = "Fresh from Spesscuba."
+	taste_message = "mojito"
```

Comment:
Mojito still has a rather specific and non-face-burning flavor. I'd rather keep it this way.